pkg/release/release.go

```diff
@@ -196,3 +201,70 @@ func URLPrefixForBucket(bucket string) string {
 	}
 	return urlPrefix
 }
+
+// extractImageManifest takes an image tar file, extracts the manifest and return the tag
+func extractTagFromManifest(tarFile string) (imageManifest string, err error) {
+	if !util.Exists(tarFile) {
+		return "", errors.New("cannot extract manifest, image path does not exist")
+	}
+	// Temp type to hold the manifest data
+	type manifestData []struct {
+		Config   string   `json:"Config"`
+		RepoTags []string `json:"RepoTags"`
+		Layers   []string `json:"Layers"`
+	}
```

Comment:
Do we wanna use this dependency instead: https://github.com/opencontainers/image-spec/blob/master/specs-go/v1/manifest.go ?

Reply:
I've been reading today the spec and sources and from what I gather, there seems to be no equivalent for the docker `RepoTags` in the OCI image spec. To list the images in the changelog, this is the data I need. Perhaps for this use case, we could do with a minimal struct. Like:
```go
manifestData []struct {
  RepoTags []string json:"RepoTags"
}
```
WDYT?  

Also, since I'm in this part of the code already I can add a function that reads the docker manifests and returns the contents as an OCI `v1.Manifest` if it's going to be needed by other parts of the anago port. I can include it in this PR if we need it.

Reply:
Sounds good, thank you for diving into this.